### PR TITLE
[HOLD, PREVIEW ONLY] StackOverflow code migration to AngularJS

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: _layouts/base.html.slim [hunk 1]
```diff
@@ -367,6 +367,7 @@
       script src="#{site.base_url}/javascripts/a-b-testing.js"
       script src="#{site.base_url}/javascripts/stackoverflow.js"
       script src="#{site.base_url}/javascripts/product-forums.js"
+
     - unless page.extra_javascripts.nil?
```
Comment (LightGuard) [inline _layouts/base.html.slim]: We need this, don't remove it.

Comment (adelasofia) [inline _layouts/base.html.slim]: @LightGuard  I didn't remove it, the red back-ground color is coming from Github's error-highlighting feature in the text editor/viewer 😃

Comment (LightGuard) [inline _layouts/base.html.slim]: Got it. That's what I get for going to be at 3 AM.

**Comment by LightGuard** [comment]
Uh, why was it closed?

**Pull request closed without merging**